package.json

```diff
@@ -65,7 +65,7 @@
   "dependencies": {
     "bson-objectid": "^1.3.1",
     "cql-exec-fhir": "^1.5.0-beta.1",
```

Comment:
How about `cql-exec-fhir`?  this has cql-execution dependency as well. Is that needs to be updated as well?

Reply:
It's a dev dependency on `cql-exec-fhir` so it shouldn't impact our usage.